index.html

```diff
@@ -23,11 +22,12 @@
 			height: 100%;
 		}
 
-		.grid {
+		/* Decorate cells for better identification */
+		#grid {
 			background: transparent;
 		}
 
-		.cell {
+		#grid>div {
```

Comment:
To avoid using extra classes in HTML we can use different selectors based on #grid for decorations